### ISSUE TITLE
ErrorCaptureStackTrace Cannot find module

### ISSUE THREAD

**Issue by drmrbrewer** [moment/luxon#1275]
I'm getting the following error when starting my node app... haven't seen this before and no idea what's causing it:

	node:internal/errors:465
		ErrorCaptureStackTrace(err);
		^

	Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/root/workspace/meteogram/node_modules/luxon/src/impl/formatter' imported from /root/workspace/meteogram/node_modules/luxon/src/info.js
		at new NodeError (node:internal/errors:372:5)
		at finalizeResolution (node:internal/modules/esm/resolve:437:11)
		at moduleResolve (node:internal/modules/esm/resolve:1009:10)
		at defaultResolve (node:internal/modules/esm/resolve:1218:11)
		at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
		at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
		at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
		at link (node:internal/modules/esm/module_job:78:36) {
	  code: 'ERR_MODULE_NOT_FOUND'
	}

Am using luxon `3.0.2`, importing via ESM:

    import { IANAZone } from 'luxon'; // https://www.npmjs.com/package/luxon

Works without problem with luxon `2.5.0`.

**Comment by herlon214** [comment]
Yeah, I also started having that error on `3.0.2`, fixing the version on `3.0.0` seems to work fine.

**Comment by drmrbrewer** [comment]
Strange, I definitely have to go back to `2.5.0` rather than `3.0.0`, but this surprises me a bit because I usually keep all my dependencies (including this one) right up to date (unless they break something) and I haven't seen this error until today.

**Comment by mrdrogdrog** [comment]
`3.0.1` works for me :shrug:

**Comment by grempe** [comment]
I just had this issue as well using Luxon with Next.js.

The following article provided the answer for me:

https://stackoverflow.com/questions/65384754/error-err-module-not-found-cannot-find-module

If I locally modified the file `node_modules/luxon/src/info.js`

And changed the import on line 6 to add the `.js` extension it resolved the issue (temporarily):

```
import Formatter from "./impl/formatter";
---
import Formatter from "./impl/formatter.js";
``` 

The offending line in the source code is here:

https://github.com/moment/luxon/blob/master/src/info.js#L6

**Comment by mrdrogdrog** [comment]
See https://github.com/moment/luxon/pull/1276

**Comment by grempe** [comment]
Tagging @icambron as this breaking change appears to have emerged in this tagged release earlier today:

https://github.com/moment/luxon/compare/3.0.1...3.0.2#diff-ed168189151bacb4f4f21b037ea4aeb8980fbc53a03aa6ba921f1e9f23fcdef0

**Comment by zdm** [comment]
Could you please make patch release?

**Comment by grempe** [comment]
It seems that @icambron released v3.0.3 to fix this.

https://github.com/moment/luxon/compare/3.0.2...3.0.3

**Comment by icambron** [comment]
Yeah, this is fixed in 3.0.3, and I marked 3.0.2 as deprecated
